src/lib/parser.py

```diff
@@ -144,6 +159,9 @@ def data_type(self) -> TokenType:
             self.eat(TokenType.KW_BOOL)
             return TokenType.BOOL
 
+        node =  DataType(token)
```

Comment:
y mo return ug `DataType` here? what happens sa code above?

Reply:
Ka kenji ni nga code, pero sa akong pa sabot. ang DataType to differentiate sa KW_ tokens og datatypes like BOOL